src/commands/dev.ts

```diff
@@ -190,6 +327,37 @@ export default class Dev extends BaseCommand {
     return healthy_services;
   }
 
+  setupTraefikServiceMap(compose: DockerComposeTemplate, gateway_port: number): Dictionary<string | undefined> {
```

Comment:
why is this showing up in the diff?

Reply:
I just moved this into its own function similar to `pollTraefik` to make the various parts of `runCompose` easier to understand

Reply:
ah thought this was the new funcs I added - ignore